4/main.js

```diff
@@ -0,0 +1,68 @@
+const sum = 0.1 + 0.3;
+console.log(sum.toFixed(1));
+const login = prompt("Введіть ваш логін.");
+const password = prompt("Введіть ваш пароль.");
+if (login == "admin") {
+  if (password == "12pass33210") {
+    console.log("Успішна авторизація!");
+  } else {
+    console.log("Пароль введено невірно.");
+  }
+} else {
+  console.log("Логін введено невірно.");
+}
+let amountofmoney = +prompt("Кількість грошей в наявності.");
+let startbudget = amountofmoney;
+console.log(
+  "Прейскурант цін: \n Watermelon: 21 грн, Carrot: 8 грн, Potato: 12 грн."
+);
+let UsersChoice = prompt("Оберіть, що ви хочете купити.");
+let pricelist = {
+  Watermelon: 21,
+  Carrot: 8,
+  Potato: 12,
+};
+if (amountofmoney < pricelist["Watermelon"]) {
+  if (amountofmoney < pricelist["Carrot"]) {
+    if (amountofmoney < pricelist["Potato"]) {
+      console.log("У вас не хватає грошей нінащо.");
```

Comment:
Too many `if` statements. What if we had 1000 products on our menu? Will that be ok for us to create 1000 `if` statements are nested? Please, pay attention